src/addons/cells/headerCells/NumberFilterableHeaderCell.js

```diff
@@ -0,0 +1,86 @@
+const React              = require('react');
+const ExcelColumn        = require('../../grids/ExcelColumn');
+
+const NumberFilterableHeaderCell = React.createClass({
+
+  propTypes: {
+    onChange: React.PropTypes.func.isRequired,
+    column: React.PropTypes.shape(ExcelColumn)
+  },
+
+  getInitialState(): {filterTerm: string} {
+    return {filterTerm: '', filterArray:[]};
+  },                                                 
+      
+  handleChange(e: Event) {
+    let val = e.target.value;
+    var tmpStrArray = val.split(",");        
```

Comment:
Please call variables something meaningful. And not a good idea to name variables after type. This should be called `filterTextInputs`